ios/Podfile.lock

```diff
@@ -341,4 +341,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce72c717ff525c8030cd40e3ac1f36f3d67fa471
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.7.4
```

Comment:
If you can, install the last version of cocoapods for not changing this file on every pull request you made, because if other developers use the 1.8.4 version then we end changing this file all time.